<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogKlien</class>
 <widget class="QWidget" name="DialogKlien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ID Klien</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editID">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nama Klien</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNama">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Alamat Klien</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editAlamat">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Telepon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editTelp">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editEmail">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Cari</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="cariKlien">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tabelKlien">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nama</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Alamat</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSimpan">
     <property name="text">
      <string>Simpan Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnUbah">
     <property name="text">
      <string>Ubah Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBatal">
     <property name="text">
      <string>Batal</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
